### ISSUE TITLE
Request: internalFetch displays results on success/failure

### ISSUE THREAD

**Issue by TiddlyTweeter** [OokTech/TW5-BobEXE#4]
internalFetch is great.

My only practical issue with it is being clear it has succeeded. I was wondering if it would be possible to display a Tiddler indicating what was successful on completion?

UPDATE: I just encountered "$:/plugins/Bob/ConflictList" that in part addresses the issue above. Unfortunately I find it confusing what to do. 

There are two aspects: 

(1) the RootWiki can display a message that differs from a slave wiki [FWIW, why is this needed at all with a fetch between two other wiki?]. 

(2) I'm unclear what to do on the options that come up when you click the individual items. 

Basically, what I need is "overwrite" for most of my use cases. Confirmation to do that would be icing on the cake, but I consider it my mistake if I get the fetch wrong, not yours. Hope this is clear.
